PHP parse scripts/Modified_php_parse_scripts_by_Baertram/DumpVars/DumpVars_main.lua

```diff
@@ -43,11 +43,7 @@ local function DumpVarsNow()
             cnt = cnt + 1
 	    end
         d(">dumped " .. tostring(cnt) .. " SOUNDS!")
-		d("DumpVars - Logout will take place in 3 seconds")
-	    zo_callLater(function()
-            d("DumpVars - Logging out now!")
-        	Logout()
-        end, 3000)
+        Logout()
```

Comment:
Do you even read my review comments? :-)
I said please do "not" use Logout as a /reloadui is totally enough to update the SV on the disk.
Why do you need to call a logout here?